tests/ViewHelpers/ApplicantFormViewHelperTest.php

```diff
@@ -15,17 +15,19 @@ public function testSuccessApplicantForm()
         ];
 
         $stageOptions = [
-            ['id' => 1, 'option' => 'dummy-option', 'stageId' => 1]
+            ['id' => 1, 'option' => 'dummy-option', 'stageId' => "1"],
```

Comment:
why the quotes here?

Reply:
we added quotes to follow the same format as the student in stages array